app/utils/kcidb.py

```diff
@@ -176,6 +179,25 @@ def push_tests(group_id, bq_options, db_options={}, db=None):
         'instance': group[models.BOARD_INSTANCE_KEY],
     }
 
+    file_name = "-".join([group[models.NAME_KEY],
+                          group[models.DEVICE_TYPE_KEY]])
+    files = [".".join([file_name, ext]) for ext in ["txt", "html"]]
+
+    lava_json = "-".join(["lava-json", group[models.DEVICE_TYPE_KEY]])
+    lava_json = ".".join([lava_json, "json"])
+    files.append(lava_json)
+
+    urls = tuple("/".join([
+        STORAGE_URL,
+        build[models.FILE_SERVER_RESOURCE_KEY],
+        file]) for file in files
+    )
+    output_files = [
```

Comment:
Using a list for the `urls` is OK but having a dictionary with the output file "names" as keys and the URLs as values would be a bit better, to more clearly map `txt` with the URL of the text log etc...

Not a big issue with just these 3 files though.

Reply:
OK, check out the latest version.  I'm using a dict for both files and url now which cleans things up better and allows for easier extending for more output files.

Reply:
Nice, thanks.